src/sds_data_model/dataframe.py

```diff
@@ -69,7 +71,28 @@
 
 @dataclass
 class DataFrameWrapper:
-    """DataFrameWrapper class.
+    """This is a thin wrapper around a Spark DataFrame.
+
+    This class stores a Spark DataFrame alongside other objects which both enhance
+    the richness of information associated with the data, and allow for increased
+    flexibility in transforming it.
+
+    Attributes:
+        name (str): The name of the dataset.
+        data (Union[SparkDataFrame, GroupedData]): Tabular data, optionally containing
+        a geometry column.
+        metadata (Metadata, optional): Object of class `Metadata` containing descriptive
+        information relating to the dataset represented in `data`.
+        lookup (Dict, optional): ???
+        graph (Digraph, optional): Object of class `Digraph` containing nodes and edges
+        relating to the source data and transformations that have taken place.
```

Comment:
Add indentation and `lookup` description.
```suggestion
        data (Union[SparkDataFrame, GroupedData]): Tabular data, optionally containing
            a geometry column.
        metadata (Metadata, optional): Object of class `Metadata` containing descriptive
            information relating to the dataset represented in `data`.
        lookup (Dict[str, Dict[Any, float]], optional): Dictionary of 
            `{column: value-map, ...}` for columns in the data. Not applied to the data. 
            Use of `categorize` will write the lookups used by the method to this
            attribute. This attribute is written to the output of `to_zarr`.
        graph (Digraph, optional): Object of class `Digraph` containing nodes and edges
            relating to the source data and transformations that have taken place.
```